lib/main.dart

```diff
@@ -49,15 +49,33 @@ class MyApp extends StatelessWidget {
           colorScheme: ThemeData.dark().copyWith().colorScheme.copyWith(
```

Comment:
Usage of `copyWith()` is redundant, you should remove it. Use copyWith only when you want to create a instance of an Object with different field values.

```suggestion
          colorScheme: ThemeData.dark().colorScheme.copyWith(
```

Reply:
Done.